public/js/main.js

```diff
@@ -15,5 +19,43 @@
                 $scope.loginUri = result.data.uri;
             });
         });
+
+        function addConversation(newConversation) {
```

Comment:
What is addConversation and getConversation meant to be doing? There are a few things about them that seem odd. First, it isn't clear why a function called getConversation should call addConversation. Also, it isn't clear why getConversation contains a post request, and why it contains logging for updating or adding a conversation. Is getConversation correctly named?